var/spack/repos/builtin/packages/voropp/package.py

```diff
@@ -23,3 +26,9 @@ def edit(self, spec, prefix):
         filter_file(r'PREFIX=/usr/local',
                     'PREFIX={0}'.format(self.prefix),
                     'config.mk')
+        if '+pic' in spec:
+            # We can safely replace the default CFLAGS which are:
+            # CFLAGS=-Wall -ansi -pedantic -O3
+            filter_file(r'CFLAGS=.*',
+                        'CFLAGS={0}'.format(self.compiler.pic_flag),
+                        'config.mk')
```

Comment:
Should we inject `spack_cxxflags`, too and also for the `~pic` case?

Reply:
Agreed. I will make these changes.